File: library_comm/src/main/java/com/dolin/comm/internal/IRecord.java
package com.dolin.comm.internal;

/**
 * @author #Suyghur.
 * Created on 2021/05/19
 */
public interface IRecord {

    void write(String msg);

    void asyncFlush();

    void expLogFile(String path, int limitSize);

    void release();
}

File: library_logger/src/main/java/com/suyghur/dolin/logger/internal/IRecord.kt
package com.suyghur.dolin.logger.internal

/**
 * @author #Suyghur.
 * Created on 4/7/21
 */
interface IRecord {

    fun write(msg: String)

    fun asyncFlush()

    fun changeLogPath(path:String)

    fun release()
}